integration-tests/openai/src/main/java/org/acme/example/openai/QuarkusRestApiResource.java

```diff
@@ -16,9 +16,7 @@
 */
 package org.acme.example.openai;
 
-import static org.acme.example.openai.MessageUtil.createChatCompletionRequest;
-import static org.acme.example.openai.MessageUtil.createCompletionRequest;
-import static org.acme.example.openai.MessageUtil.createEmbeddingRequest;
+import static org.acme.example.openai.MessageUtil.*;
```

Comment:
I'd prefer we don't use star imports

Reply:
I ran `mvn impsort:sort` from the project root and thats what it produced. If I commit without running that then the build fails saying I need to run it...